controls/stig_rhel9.yml

```diff
@@ -3119,14 +3128,6 @@ controls:
             - require_singleuser_auth
         status: automated
 
-    -   id: RHEL-09-611205
-        levels:
-            - medium
-        title: RHEL 9 must prevent system daemons from using Kerberos for authentication.
-        rules:
-            - kerberos_disable_no_keytab
```

Comment:
Add this rule to RHEL 9 default profile to prevent its removal from RHEL 9 data stream.